Sofa/framework/Core/src/sofa/core/ConstraintParams.h

```diff
@@ -121,39 +115,39 @@ class SOFA_CORE_API ConstraintParams : public sofa::core::ExecParams
 
 
 
-    /// @name Access to vectors from a given SingleLink to a state container (i.e. State or MechanicalState)
-    /// @{
+//    /// @name Access to vectors from a given SingleLink to a state container (i.e. State or MechanicalState)
```

Comment:
Is the plan to wait for the CI and then deprecate those functions?

Reply:
For the moment I was more about first having discussion if we want to do this move. 

If everyone think this is a good move to minimize API's surface then we should go for the official deprecation-disable-remove process with this timeline:
- deprecation (from now to 22.12 release)
- disable (from 22.12 at least up to to 23.12)  as I assume the API is used in several plugin, people will have to update quickly.
- finally remove